aws/src/main/java/org/apache/iceberg/aws/s3/signer/S3ObjectMapper.java

```diff
@@ -48,13 +48,16 @@ public class S3ObjectMapper {
 
   private S3ObjectMapper() {}
 
-  static ObjectMapper mapper() {
+  public static ObjectMapper mapper() {
```

Comment:
Does the underlying object mapper need to be public? It feels like that shouldn't be if this `S3ObjectMapper` class is supposed to expose mapper operations

Reply:
you're right, making `initModule()` public is usually sufficient and should be enough. It just felt weird to leave this non-public while making `initModule()` public. I've reverted this change, since having public access to `initModule()` works for our use case.